src/Restaurant.WebApi/Controllers/TableController.cs

```diff
@@ -14,21 +14,24 @@ public class TableController(ITableService tableService) : Controller
     {
         private readonly ITableService _tableService = tableService;
 
-
-        // [Authorize(Roles = UserRoles.Admin)]
+        //[Authorize(Roles = UserRoles.Admin)]
```

Comment:
Make sure that you will fix this code in your next PR. Usually, you should put some kind of tag to easily finding it